extensions_userprovided/raspiBackupDiscordWrapper.sh

```diff
@@ -0,0 +1,241 @@
+#!/bin/bash
+
+#######################################################################################################################
+#
+# 	Sample script to wrap raspiBackup.sh in order to send Discord notification message once raspiBackup is finished
+#	Tested with raspiBackup.sh version: 0.6.8 CommitSHA: 120287b CommitDate: 2022-12-15 CommitTime: 16:48:38
+#	relies on raspiBackup variables:
+#	  # BACKUP_TARGETDIR refers to the backupdirectory just created. Used when raspiBackup succeeded to read the raspiBackup.msg file content
+#	In case raspiBakup failed, the file raspiBackup.msg file under HOME folder of calling user is used
+#
+# 	Visit http://www.linux-tips-and-tricks.de/raspiBackup for details about raspiBackup
+#
+#	NOTE: This is sample code how to extend functionality of raspiBackup and is provided as is with no support.
+#
+#	--> Requires:
+#		curl (https://curl.se):
+#			for sending POST command to Discord WebHook
+#			Tested with version: curl 7.64.0
+#		jq (https://stedolan.github.io/jq/):
+#			For JSON manipulation in shell
+#			Tested with version jq-1.5-1-a5b5cbe
+#
+#	--> Configuration:
+# 		-> file: /usr/local/etc/discordWrapper.conf
+#		-> Content of the config file:
+#	 	# URL of the Discord WebHook where to send the message. Adapt to your case
+#		CONFIG_DISCORD_URL="https://discord.com/api/webhooks/1234567890.../abcdef...."
+#
+#		# Possible Colors to be used for Discord Embed message. Must be integer value in base 10 for RGB clolor coding. Refer to Discord documentation (https://discord.com/developers/docs/intro)
+# 		COLOR_BLUE=255
+#		COLOR_RED=16711680
+#		COLOR_GREEN=65280
+#
+# 		# Color to be used depending on the raspiBackup execution result: success or failure
+#	        # each line shall refer to one of the color defined here upper
+# 		CONFIG_COLOR_DEFAULT=$COLOR_BLUE
+# 		CONFIG_COLOR_SUCCESS=$COLOR_GREEN
+# 		CONFIG_COLOR_FAILURE=$COLOR_RED
+#
+#
+#######################################################################################################################
+#
+#   Copyright (c) 2013-2022 framp at linux-tips-and-tricks dot de
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#######################################################################################################################
+
+set -euf -o pipefail
+
+MYSELF=${0##*/}
```

Comment:
The whole git handling stuff will work only if the code will be published in a special way I use to publish raspiBackup parts. You can keep the code - but you ma just use `VERSION`

Reply:
should I remove `set -euf -o pipefail` ?

Reply:
done